app/views/pipeline_jobs/index.html.erb

```diff
@@ -67,7 +67,7 @@
                ) %>
 
               <tr>
-                <td><%= report.harvest_job.name %></td>
+                <td><%= report.definition_name || report.harvest_job.name %></td>
```

Comment:
question: should the definition name always be available and therefore we could remove the ||?